lib/puppet/file_serving/content.rb

```diff
@@ -30,7 +30,7 @@ def collect(source_permissions = nil)
   def content
     unless @content
       # This stat can raise an exception, too.
-      raise(ArgumentError, "Cannot read the contents of links unless following links") if stat.ftype == "symlink"
+      raise(ArgumentError, _("Cannot read the contents of links unless following links")) if stat.ftype == _("symlink")
```

Comment:
I don't think "symlink" should be wrapped here - that's going to be a string returned by ruby